pkg/endpoint/local_ip.go

```diff
@@ -37,3 +39,15 @@ func GetLocalIPForDestination(dst string) string {
 func GetLocalIP() string {
 	return GetLocalIPForDestination("8.8.8.8")
 }
+
+func GetLocalIPs(ipFamilies []v1.IPFamily) []string {
```

Comment:
Since there can be at most 2 families:

```suggestion
func GetLocalIPs(ipFamilies [2]v1.IPFamily) []string {
```